app/models/concerns/mentionable.rb

```diff
@@ -46,4 +46,22 @@ def parse_id_from_mention_text
   def parse_id_from_decorated_ids(ids)
     ids.map { |id| id.match(/\d+/).to_s }
   end
+
+  included do
+    def self.mentionable_users(opts = {})
+      users = active.exclude_user(User.current_user.id)
+                    .with_roles(mentionable_roles(opts[:roles])).distinct.to_a
+      users << add_order_creator(users, opts[:order_id]) if opts[:order_id]
+      users.compact
+    end
```

Comment:
It's usually preferable to use ruby options instead of a hash, that way it's much clearer
```ruby
    def self.mentionable_users(roles:, order_id: nil)
      users = active.exclude_user(User.current_user.id)
                    .with_roles(mentionable_roles(roles)).distinct.to_a
      users << add_order_creator(users, order_id) if order_id
      users.compact
    end
```

Not a big deal, but now we know what the parameters are and which ones are optional by looking at the function signature